### PR TITLE
added POST feature and did some error fixing

### PR DESCRIPTION
**Pull request by benmaier**
I've added functionality to use an HTTP request (POST) to transfer the computed styling back to Python.

I've also declared some previously undeclared variables which used to caused errors on my machine.

### CODE REVIEW

**Comment by ulfaslak** [comment]
Styling as in the config? Or the image? Would be aaaaawesome if we could pipe the canvas image back into a notebook as cell output

**Comment by ulfaslak** [comment]
Ah just saw the issue on netwulf I understand now.

**Comment by benmaier** [comment]
Would be cool to pipe the image back ... I think I know how to approach this in general, but it'd take some time :(

File: vis.js [hunk 1]
```diff
@@ -936,6 +938,7 @@ function vis(new_controls) {
 
   var hoveredNode;
   var selectedNodes = [];
+    console.log(selectedNodes);
```
Comment (benmaier) [inline vis.js]: This should be removed again. sorry :(

**Comment by ulfaslak** [comment]
Is there a way for the program to tell if it was launched from a Python instance? Since I still want there to be a web version (because I use that quite a lot), this should of course not have the POST feature.

**Comment by benmaier** [comment]
We could add an option to the url, like `?python=true` or smth?

**Comment by ulfaslak** [comment]
Good idea! Seems like `location.href` is what I was looking for. I will implement this.

**Pull request merged**